person.rb

```diff
@@ -19,3 +26,39 @@ def of_age?
     false
   end
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] Here you should capitalize the first letter of the given name, kindly make sure to use `correct_name.capitalize` instead of `correct_name.upcase`